### PR TITLE
Don't attempt to set context on non existant paths

### PR DESCRIPTION
**Pull request by sdodson**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1564840

Corrects problem introduced in https://github.com/openshift/openshift-ansible/pull/7466

### CODE REVIEW

File: roles/container_runtime/tasks/package_crio.yml [hunk 1]
```diff
@@ -70,10 +70,17 @@
     dest: /etc/sysconfig/crio-network
     src: crio-network.j2
 
+- name: Check for /var/lib/containers/overlay2
```
Comment (michaelgugino) [inline roles/container_runtime/tasks/package_crio.yml]: When will this be present or not present, that's what's not clear?  I don't like having this here if it's not useful in all cases.

I would like to hear from whoever decided we needed this in the first place.

File: roles/container_runtime/tasks/package_crio.yml [hunk 2]
```diff
@@ -70,10 +70,17 @@
...
+  stat:
+    path: /var/lib/containers/overlay2
+  register: varlibcontainers
+
+# TODO: Use selinux modules
 - name: Fixup SELinux permissions for docker
```
Comment (michaelgugino) [inline roles/container_runtime/tasks/package_crio.yml]: We should fixup this to not use shell, just issue two commands.

**Comment by sdodson** [comment]
We'll see how this tests out without it.

**Comment by sdodson** [comment]
nothing works :-(
/retest

**Pull request merged**